### DISCUSSION TITLE
Alt account verify

### DISCUSSION THREAD

**Discussion by NEON-CYC** [Vedza/NitroSniperGo#162] [Q&A]
Does the sniper stop working if they ask your alt to verify using a number ?

**Comment by Vedza** [comment]
I have no idea as it never happened to me, but I guess alt will stop sniping.

**Reply by DarkGD123** [reply]
Yeah discord asks on My 11 alts for numbers but they are  working normally, but cant snipe a giveaway

**Reply by ghost** [reply]
you can snipe giveaways and snipe nitro, but no auto-dm

**Comment by TacoSnipesYou** [comment]
Old discussion, but no I have multiple accounts that require phone verification that still join giveaways and redeem nitros
